### PR TITLE
rename config to laravel-handlebars

### PR DESCRIPTION
**Pull request by greut**
Addresses #24

### CODE REVIEW

**Comment by Lorti** [comment]
This does not break any existing installations, if I understand correctly?

File: README.md [hunk 1]
```diff
@@ -25,13 +25,16 @@ Laravel Handlebars is distributed as a composer package. So you first have to ad
 Then you have to run `composer update` to install the package. Once this is completed, you have to add the service provider to the providers array in `config/app.php`:
 
 ```
-'ProAI\Handlebars\HandlebarsServiceProvider'
+        /*
+         * Package Service Providers...
```
Comment (Lorti) [inline README.md:29]: Do we need the indentation? 🙂

File: README.md [hunk 2]
```diff
@@ -25,13 +25,16 @@ Laravel Handlebars is distributed as a composer package. So you first have to ad
...
+         */
+        ProAI\Handlebars\HandlebarsServiceProvider::class,
 ```
 
 You can publish the package configuration with the following command:
 
-```
-php artisan vendor:publish --provider="ProAI\Handlebars\HandlebarsServiceProvider"
+```console
+php artisan vendor:publish --tag=laravel-handlebars
```
Comment (Lorti) [inline README.md]: If I don't specify a tag when calling the `php artisan vendor:publish` command, will it still publish the config for this package?

**Comment by greut** [comment]
if you don't specify anything, it asks you what to do. Renaming displays `laravel-handlebars` instead of `config`. That's about it.

**Review by Lorti — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (greut) [reply README.md:29]: Copy and paste from the file itself.

**Comment by Lorti** [comment]
Thanks, I'll publish a new version!

Resolves #24

**Pull request merged**